Vagrantfile

```diff
@@ -672,6 +672,32 @@ fi
 jane notify info "Ansible Controller provisioning complete"
 SCRIPT
 
+$setup_examples = <<SCRIPT
+set -o nounset -o pipefail -o errexit
+
+projects_dir=$HOME/src
+controller_dir=${projects_dir}/controller
+upstream_dir=/vagrant/lib/examples
+examples=$(find /vagrant/lib/examples/* -maxdepth 0 -type d -printf "%f\n")
+
+# controller_dir will be our source of truth
+rsync -a ${upstream_dir}/bootstrap/ansible/ ${controller_dir}/ansible/
+jane notify info "${upstream_dir}/bootstrap merged into ${controller_dir}"
+
+# bootstrap_dir is resynchronized with controller one time just to get
+# auto-generated inventory
+for example in ${examples}; do
+    rsync -a ${upstream_dir}/${example} ${projects_dir}
+    jane notify info "${upstream_dir}/${example} installed in ${projects_dir}"
+    rsync -a ${controller_dir}/ ${projects_dir}/${example}/
+    jane notify info "${projects_dir}/${example}/ synchronized with ${controller_dir}/"
+done
+jane notify success "All examples installed and synchronized in ${projects_dir}"
+
+wget -q -O $HOME/.ssh/id_rsa.pub https://raw.githubusercontent.com/hashicorp/vagrant/master/keys/vagrant.pub
+jane notify info "Insecure vagrant **public** key installed in $HOME/.ssh for bootstrap example"
```

Comment:
Is this really needed? The current `Vagrantfile` should alrady provision the insecure Vagrant key, it's necessary for communication between the nodes.

Reply:
I would like to avoid this in order to keep examples independent from Vagrant setup.

Current vagrant provisioning provision vagrant insecure **private** key in `/home/vagrant/.ssh`, not public key. Consequently, when you want to bootstrap a host with another user thant `ansible-admin`,  `debops.system_users` will not import any SSH key in `/home/ansible-admin/.ssh`

Reply:
The insecure public key is available in `/home/vagrant/.ssh/authorized_keys`, you should be able to grab it from there if the provision script is executed as the `vagrant` user - no need to hit an eternal URL every time a Vagrant VM is provisioned. Just copy the `authorized_keys` file to `id_rsa.pub` and you should be fine.

Reply:
Nice idea, I did it.

However, in current `Vagrantfile`, this setup will work only if you are running `debian/buster64` box:

```ruby
        if ENV['VAGRANT_BOX'] || 'debian/buster64' == 'debian/buster64'
            subconfig.ssh.insert_key = false
```
From my point of view, there is no need for such test for master or nodes box.

Any reason about this current config ?

Reply:
I think we can drop this one, good catch. It will also help when `debian/stretch64` is used in some CI tests. I'll check it out in the CI pipeline to be sure.